pkg/gofr/http/router.go

```diff
@@ -13,6 +13,8 @@ type Router struct {
 	RegisteredRoutes *[]string
 }
 
+type Config func(r *Router)
```

Comment:
this function seems to be unused!

Reply:
I was thinking of some usecases but first we will push this static handler into the framework and later work on the enhancements. I will remove this line of code in next commit.